app/views/availabilities/_form.html.erb

```diff
@@ -3,14 +3,11 @@
 
   <div class="form-inputs">
     <%= f.association :person, collection: Person.active %>
-    <%= f.input :status, :as => :select,
-                :collection => (Availability::STATUS - ["Cancelled"]) %>
+    <%= f.input :status, :as => :select, :collection => (Availability::STATUS - ["Cancelled"]) %>
     <%= f.input :description %>
 
-    <%= f.input :start_time, :as => :string,
-                :input_html => { :class => 'datetimepicker', :size => 11 }  %>
-    <%= f.input :end_time, :as => :string,
-                :input_html => { :class => 'datetimepicker', :size => 11 }  %>
+    <%= f.input :start_time, as: :string, input_html: { class: 'datetimepicker', size: 11, value: format_datetime_value(f.object.start_time) } %>
+    <%= f.input :end_time, as: :string, input_html: { class: 'datetimepicker', size: 11, value: format_datetime_value(f.object.end_time) } %>
```

Comment:
These are very long lines, which do not fit into the default github UI. Please break them across two lines, similar to the originals. On the plus side, yours are more accurate !